### ISSUE TITLE
Execution failed for task ':rive_common:buildCMakeRelWithDebInfo[arm64-v8a]'.

### ISSUE THREAD

**Issue by sydney1232** [marcos930807/awesomeDialogs#140]
I just recently encountered this issue. I am certain that it came from AwesomeDialogs. 

I am not using Rive package. 

Please fix asap.
![image](https://github.com/marcos930807/awesomeDialogs/assets/157961996/c07ed4e1-9d5f-4234-9897-0df4dc8c3770)

![image](https://github.com/marcos930807/awesomeDialogs/assets/157961996/60e24c17-97d3-4b04-bf80-abc067d2ce78)

**Comment by AmjadArbasi** [comment]
![image](https://github.com/marcos930807/awesomeDialogs/assets/111657853/06c1f564-aa88-4a6b-ae23-17f57932cbbc)

I have same problem

**Comment by PoliteCoder97** [comment]
same issue

**Comment by mfurkanyuceal** [comment]
Same issue

**Comment by oguzarapkirli** [comment]
yeah same issue for 1-2 years. every time i opened an old project i am finding myself here

**Comment by marcos930807** [comment]
Be aware of this https://github.com/rive-app/rive-flutter/issues/385 Rive is becoming a nightmare, missing flare stability in the old days. After upgrade rive rive comon 0.4.7 has problem override to 0.4.5 in the meantime.

**Comment by marcos930807** [comment]
dependency_overrides:
  rive_common: 0.4.5

**Comment by marcos930807** [comment]
> yeah same issue for 1-2 years. every time i opened an old project i am finding myself here

Very helpfull comment.

**Comment by mfurkanyuceal** [comment]
@marcos930807 You can look at this solution.
https://stackoverflow.com/a/78433531/15446019

**Comment by marcos930807** [comment]
> @marcos930807 You can look at this solution. https://stackoverflow.com/a/78433531/15446019

yea, override the rive package override the rive common that has the problem, so it temporally fix the issue. I am not fan of this override aproach but it is what it is. I can't control rive package but be asure that when i find a specific version that works good i will never update again., i recently did an upgrade trying to solve a problem with android binary size and then reaper this issue. Rive is becoming a nightmare sadly is a non removable dependency right now. Thanks

**Comment by korkis** [comment]
In my case awesome_dialog: ^3.2.0 version works

**Comment by gi-rubyl** [comment]
I encountered this exact issue while building on Android, and I have a solution that worked for me.

The core problem appears to be a C++ build system configuration failure within the rive_native plugin (specifically, CMake couldn't locate the dart executable). Even after running standard Flutter/Dart clean commands, the native build cache remained corrupted.

| Component | Version |
| -------- | ------- |
| Flutter  | 3.29.3    |
| Dart | 3.7.2   |
| Android NDK    | 27.0.12077973   |

The standard methods (`flutter clean`, `dart pub cache clean`, `dart pub cache repair`) did not work for me. Then, I tried to run the build directly via Gradle by running the commands below:
`./gradlew clean`
`./gradlew assembleDebug`

After running these commands, I went back to the project root and ran `flutter run`, and the build succeeded without error. I believe this forces the C++ build system (CMake/NDK) to reset its cache and properly find all the required paths, including the Dart SDK path.

**Comment by igorgoncalves** [comment]
> I encountered this exact issue while building on Android, and I have a solution that worked for me.
> 
> The core problem appears to be a C++ build system configuration failure within the rive_native plugin (specifically, CMake couldn't locate the dart executable). Even after running standard Flutter/Dart clean commands, the native build cache remained corrupted.
> Component 	Version
> Flutter 	3.29.3
> Dart 	3.7.2
> Android NDK 	27.0.12077973
> 
> The standard methods (`flutter clean`, `dart pub cache clean`, `dart pub cache repair`) did not work for me. Then, I tried to run the build directly via Gradle by running the commands below: `./gradlew clean` `./gradlew assembleDebug`
> 
> After running these commands, I went back to the project root and ran `flutter run`, and the build succeeded without error. I believe this forces the C++ build system (CMake/NDK) to reset its cache and properly find all the required paths, including the Dart SDK path.

This really work to me. Thanks!!!
